### PR TITLE
[ONNX] LRN

### PR DESCRIPTION
**Pull request by cynthia-liu**

### CODE REVIEW

File: pyzoo/zoo/pipeline/api/onnx/mapper/lrn.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    def __init__(self, node, _params, _all_tensors):
+        super(LRNMapper, self).__init__(node, _params, _all_tensors)
+
+    def _to_tensor(self):
+        data = self.model_inputs[0].zvalue
+        alpha = float(self.onnx_attr['alpha']) if "alpha" in self.onnx_attr else 0.0001
+        beta = float(self.onnx_attr['beta']) if "beta" in self.onnx_attr else 0.75
+        bias = float(self.onnx_attr['bias']) if "bias" in self.onnx_attr else 1.0
+        size = int(self.onnx_attr['size'])
+        dim_ordering = "th"
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/mapper/lrn.py]: what if dim_ordering is not "th"? you should throw exception here.

Comment (cynthia-liu) [reply pyzoo/zoo/pipeline/api/onnx/mapper/lrn.py]: LRN in ONNX doesn't have attribute dim_ordering, so I guess we need to hardcode "dim_ordering = 'th'"

**Comment by cynthia-liu** [comment]
LRN is needed in ONNX models: inceptionv1, Googlenet, Zfnet512

**Comment by zhichao-li** [comment]
let's rebase and merge this.

**Comment by cynthia-liu** [comment]
ZOO-PR-Validation - Build # 2047 - Fixed:

Check console output at http://10.239.47.211:18888/job/ZOO-PR-Validation/2047/ to view the results.

The pull request: https://github.com/intel-analytics/analytics-zoo/pull/1077 Fixed

**Pull request merged**